actors/miner/src/lib.rs

```diff
@@ -4464,25 +4460,6 @@ fn handle_proving_deadline(
     let state: State = rt.transaction(|state: &mut State, rt| {
         let policy = rt.policy();
 
-        // Vesting rewards for a miner are quantized to every 12 hours and we can determine what those "vesting epochs" are.
```

Comment:
Moved to the end, see the comment.

Reply:
> the benefit of moving this below

because it's done automatically as part of paying the fee, then it's done a second time _just in case_